index.html

```diff
@@ -1851,39 +1851,279 @@ <h5><code>invokeaction</code></h5>
           </a>).
         </section>
       </section>
+
       <section id="events">
         <h4>Events</h4>
-        <section class="ednote">
+        <p>
+          The Core Profile uses Server-Sent Events [[EVENTSOURCE]] as a
+          mechanism for Consumers to subscribe to events emitted by a Web Thing.
+          Any <code>Form</code> within an <code>Event</code> affordance whose 
+          <code>href</code> member contains a URI with a URI scheme of 
+          <code>http</code> or <code>https</code> is assumed to have a 
+          <code>subprotocol</code> of <code>sse</code>, unless specified
+          otherwise.
+        </p>
+        <section id="subscribeevent">
+          <h5><code>subscribeevent</code></h5>
+          <p class="rfc2119-assertion" 
+            id="core-profile-protocol-binding-events-subscribeevent-1">
+            The URL of an <code>Event</code> resource to be used when 
+            subscribing to an event MUST be obtained from a Canonical TD by
```

Comment:
Let's discuss why you need canonical TDs here.   The canonical form should not change URLs - perhaps you meant the *form*, e.g. with default values filled in?

Reply:
The important part here is that defaults need to be applied to the `Form`, otherwise there may not be a `Form` with a `subscribeevent` operation because the `op` member may have been omitted. Specifying _Canonical_ TD is just a shorthand way of saying to apply defaults first. Please let me know if you can think of a better way of specifying that.

Reply:
Probably safer to just explictly say "apply all defaults" and not depend on canonical TDs